docs/getting-started/educator-pipeline/overview.md

```diff
@@ -0,0 +1,89 @@
+
+### Current State
+
+![EPDM Current State](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_current.png)
+
+## Which is best for my organization - EPDM-Core or EPDM-Community?
+
+| Choose EPDM-Core if... | Choose EPDM-Community if... |
+|------------------------|----------------------------|
+| I want to get started as quickly and simply as possible – using Ed-Fi EPP Starter Kits | I am currently using, or plan to use EPDM entities that are not currently in EPDM-Core |
+| My use cases focus on candidate demographics, candidate enrollment, or performance (candidate or others) | |
+
+If you are upgrading from TPDM v1.0 (and not using starter kits), or are unsure, start with TPDM-Community.
+
+## Installing EPDM
+
+| Installing EPDM-CORE | Installing EPDM-Community |
+|----------------------|---------------------------|
+| Install the Ed-Fi ODS/API platform v5.3 (defaults to including EPDM-Core extension) | EPDM-Community: Install the Ed-Fi ODS/API platform v5.3, but change the EPDM extension to be installed to EPDM-Community. This is done by following these [installation instructions](https://edfi.atlassian.net/wiki/spaces/EPP/pages/23169945/Installing+Ed-Fi+ODS+API+5.3+with+EPDM-Community+v1.1). |
+
+### Visual Representation
+
+#### EPDM-Core
+
+![EPDM-Core](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_core.png)
+
+#### EPDM-Community
+
+![EPDM-Community](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_community.png)
+
+More details about [TPDM-Core](https://edfi.atlassian.net/wiki/display/EFDS33/Educator+Preparation+Data+Model+Domain) and [TPDM-Community](https://edfi.atlassian.net/wiki/display/TPDMX/TPDM+Community?src=contextnavpagetreemode).
+
+## ​Changing the EPDM model installed
+
+If you start with EPDM-Core, and decide later that you need EPDM-Community, you can make the change by installing a new instance of the ODS/API and following the install instructions.​
+
+If you have already loaded data, the choices are:​
+
+You can re-load data into the ODS/API without changes (since EPDM-Core is a strict subset) ​
+If data can’t be reloaded for some reason, it would be possible to write database scripts to migrate data – but this would incur some additional time/cost​
+
+## Future Plans
+
+The Alliance plans to continue to work with the EPP community to identify further entities that are mature enough to move into Ed-Fi Core.
+
+The following are resources for getting information and engaging in the process:
+
+* [Ed-Fi Community Governance](https://edfi.atlassian.net/wiki/display/GOV/Ed-Fi+Governance) – for convening the community and sharing information.
+* [Ed-Fi Technology Roadmap](https://edfi.atlassian.net/wiki/display/ETKB/Ed-Fi+Technology+Roadmap) - for details on upcoming releases.
+* [Ed-Fi Tracker](https://tracker.ed-fi.org/secure/RapidBoard.jspa?rapidView=175) – for submitting tickets.
```

Comment:
Replace with a link to the [Ed-Fi Community Hub](https://community.ed-fi.org) for submitting cases when people have a question or suggestion about how to use the Educator Preparation Data Model.